firefox-ios/Client/Frontend/Autofill/Address/Edit/Address+Encodable.swift

```diff
@@ -0,0 +1,63 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Storage
+
+extension CodingUserInfoKey {
+    static let formatStyleKey = CodingUserInfoKey(rawValue: "FormatStyleKey")!
+}
+
+enum FormatStyle {
+    case kebabCase
+}
+
+extension Address: Encodable {
```

Comment:
This is the A~S address right?

Reply:
hey @nbhasin2. What A~S means?

Reply:
Application Service @razvanlitianu

Reply:
Correct. Here I'm not sure what is the best solution. Ideally we would have our own Address model, but since we have use the AS one on the other implementations I chose to stick with it and conform it to Encodable.